plugins/plugin-sass/plugin.js

```diff
@@ -71,9 +71,9 @@ module.exports = function sassPlugin(_, {native, compilerOptions = {}} = {}) {
       // check no ext: "_index" (/a/b/c/foo/_index)
       this._markImportersAsChanged(filePathNoExt);
       // check no underscore: "index.scss" (/a/b/c/foo/index.scss)
-      this._markImportersAsChanged(filePath.replace(/([\\\/])_/, '$1'));
+      this._markImportersAsChanged(filePath.replace(/(.*[\\\/])_/, '$1'));
       // check no ext, no underscore: "index" (/a/b/c/foo/index)
-      this._markImportersAsChanged(filePathNoExt.replace(/([\\\/])_/, '$1'));
+      this._markImportersAsChanged(filePathNoExt.replace(/(.*[\\\/])_/, '$1'));
```

Comment:
bah, just realized that this will match this:

```
 /foo/_Playground/snowpack-test/src/base.sass
```

Going to replace with something closer to your original and then merge: `\/_(?!.*\/)`

```
✅  /Users/fks/Downloads/_Playground_/snowpack-test/src/_base.sass
✅  /Users/fks/Downloads/_Playground/snowpack-test/src/_base.sass
✅  /Users/fks/Downloads/Playground/snowpack-test/src/_base.sass
🆇  /Users/fks/Downloads/_Playground/snowpack-test/src/base.sass
```